src/connector/src/source/nexmark/mod.rs

```diff
@@ -12,21 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-pub mod config;
 pub mod enumerator;
 pub mod source;
 pub mod split;
-mod utils;
 
 use std::collections::HashMap;
 
 pub use enumerator::*;
+use nexmark::config::{NexmarkConfig, RateShape};
 use serde::Deserialize;
 use serde_with::{serde_as, DisplayFromStr};
 pub use split::*;
 
-const NEXMARK_BASE_TIME: usize = 1_436_918_400_000;
-
 pub const NEXMARK_CONNECTOR: &str = "nexmark";
 
 const fn identity_i32<const V: i32>() -> i32 {
```

Comment:
Can we directly use `core::convert::identity` for this? 🤣

Reply:
Seems not. This utilizes const generic but `identity` doesn't have it.